local-kubernetes/run

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
+set -x
+
+# Make sure we're in the right directory
+cd $(cd -P -- "$(dirname -- "$0")" && pwd -P)
+
+export KUBERNETES_PROVIDER=vagrant
+export KUBERNETES_MEMORY=512
+
+if [ ! -d kubernetes ]; then
```

Comment:
That's testing whether the `kubernetes` directory exists.
If it doesn't exist, the get.k8s.io script will create it.